update_validator.sh

```diff
@@ -31,18 +31,18 @@ source_script "update_env.sh"
 # Navigate to the validation directory and source update script
 
 cd "../validation" || exit 1
-source_script "update_env.sh"
```

Comment:
As far as I understand, running this only rstarts the validator process. Is that what you want ?

Reply:
It updates the env. Restart is performed on the last line of script